installer/templates/new/assets/javascripts/application.js

```diff
@@ -27,13 +27,35 @@ class Kitto {
     content_width = (widget_base_dimensions[0] + widget_margins[0] * 2) * columns;
 
     $gridster.width(content_width);
-    $('.gridster ul:first').gridster({
+    $('.gridster > ul').gridster({
       widget_margins: widget_margins,
       widget_base_dimensions: widget_base_dimensions
     });
 
     return this;
   }
+
+  static initializeRotator() {
+    let $rotator = $('rotator');
+    let $dashboards = $rotator.children();
+
+    if ($dashboards.length < 2) { return this; }
+
+    let dashboard_count = $dashboards.length;
+    let interval = $rotator.data('refresh') * 1000;
+    let current_dashboard_index = 0;
+
+    let rotate = () => {
+      $dashboards.hide();
+      $($dashboards[current_dashboard_index]).show();
+
+      current_dashboard_index = (current_dashboard_index + 1) % dashboard_count;
```

Comment:
Identifier 'current_dashboard_index' is not in camel case.

Enforce the usage of camel case to have a consistent codebase.

[Read more about it here.](http://eslint.org/docs/rules/camelcase)